src/Scriban.Tests/TestFiles/100-expressions/111-binary-compare.txt

```diff
@@ -47,3 +47,9 @@ a == empty: {{
     a == empty
 }}
 b == empty: {{b == empty}}
+
```

Comment:
Remove this extra line. This is what is failing the roundtrip test

Reply:
I've been removed the trail spaces in test files, but unfortunately I have no luck with run it on my machine. NUnit cant discover this tests.  I don't have any experience with NUnit so I hope for you help.
`
 No test matches the given testcase filter `FullyQualifiedName=Scriban.Tests.TestParser.A100_expressions` in C:\Projects\VirtoCommerce\scriban\src\Scriban.Tests\bin\Debug\netcoreapp2.1\Scriban.Tests.dll
`

And `AsyncCodeGen ` generate invalid code. See screenshot in message above.